server/routes/core.ts

```diff
@@ -88,18 +88,27 @@ router.get('/sitemap.txt', async (_, res: Response) => {
  * The current owners are listed below.
  * For each owner, please add github name and verification file.
  */
+const verificationFiles: Array<{ owner: string; filename: string }> = [
+	{
+		owner: 'kelvin-chappell',
+		filename: 'google6e3510e8603d6b4c.html',
+	},
+	{
+		owner: 'john-duffell',
```

Comment:
what's the reason to put the owner as a string instead of a comment?  it might be a lot more concise to just have a string array with a comment after each line

Reply:
hmm, I'm easy either way, I guess this way it forces you to add your name in there or typescript will moan at you